### ISSUE TITLE
Social Media Links Need Labeling

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1789]
### Prerequisites

* [X] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [X] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [X] Are you experiencing this issue with one of our built-in root or book themes?
* [X] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Steps to Reproduce

1. Navigate to a [book page](https://uw.pressbooks.pub/lafrancesauvee/)
2. Navigate to the link with a twitter icon on it directly after the "Read Book" link
3. Inspect and notice there is no labeling or alt-text for the svg

**Expected behavior:** 
The icon should be given alt-text or the link should be given a label.
**Actual behavior:** 
It does not have alt-text or any kind of label.

**Comment by SteelWagstaff** [comment]
Lots of accessbility patterns for SVGs discussed here: https://www.deque.com/blog/creating-accessible-svgs/. Two best options appear to be `<img>` tags with alt text and `src=svg link` OR `<svg> + role="img" + <title> + <desc> + aria-labelledby="[ID]"`

**Comment by avaaec** [comment]
This is a UW ATS priority 2 issue

**Comment by connermwallace** [comment]
UW Accessible Technology Services has inspected this issue and found that the twitter icon is not navigable to by keyboard. This was tested on Chrome Version 79.0.3945.88 (Official Build) (64-bit) and Firefox 68.3.0esr (64-bit) with Pressbooks Version: 5.3.2.
